### DISCUSSION TITLE
Indoor Camera Settings for HomeKit Automation States

### DISCUSSION THREAD

**Discussion by jglazner** [samemory/homebridge-eufy-security-control#333] [Q&A]
I'm not quite sure how I'm supposed to use the HomeKit modes for cameras to change the state.  I see in the configuration there is Home, Away, Night, and Off settings.  For my indoor pan and tilt camera's I would like to turn off notifications etc during the day, but get alerts and what not at night.  Is something like this possible?

**Comment by schliemann** [comment]
@jglazner When you write alert do you then mean alarm?
